skopt/utils.py

```diff
@@ -292,3 +292,75 @@ def cook_estimator(base_estimator, space=None, **kwargs):
 
     base_estimator.set_params(**kwargs)
     return base_estimator
+
+
+def dims_aslist(params_space):
+    """Convert dictionary of parameter names : skopt.space.Dimension
+    a list of skopt.space.Dimension, that can be used in Optimizer.
```

Comment:
omg lol yeah it was late when I was writing it :D
I will use your version - thanks for coming up with this!